plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/add/AddToIndexViewImpl.java

```diff
@@ -101,22 +96,8 @@ protected void onEnterClicked() {
 
     /** {@inheritDoc} */
     @Override
-    public void setMessage(@NotNull String message, @NotNull List<String> items) {
-        this.message.setHTML(message);
-        if (items == null || items.isEmpty()) {
-            this.items.setVisible(false);
-            this.items.setText("");
-        } else {
-            this.items.setVisible(true);
-            final StringBuilder sb = new StringBuilder();
-            String toAppend = "";
-            for (String item : items) {
-                sb.append(toAppend);
-                toAppend = "\n";
-                sb.append(item);
-            }
-            this.items.setText(sb.toString());
-        }
+    public void setMessage(String message) {
+        this.message.getElement().setInnerHTML(message);
```

Comment:
The `message` parameter is given in html format, and `setText()` would display the raw massage with html tags. How about just to rename this parameter to `htmlMessage`?